Assets/_Scripts/StageManager.cs

```diff
@@ -145,7 +150,7 @@ private void CheckIsFull(StagePlacement placement)
         {
             //If an empty spot (and turned on!), then its not full.
             (bool, bool) pointStatus = placementPoint.IsOccupied();
-            if (pointStatus is {Item1: false, Item2: false} && placementPoint.gameObject.activeSelf)
+            if ((!pointStatus.Item1 || !pointStatus.Item2) && placementPoint.gameObject.activeSelf)
```

Comment:
This one-liner might be able to be simplified?

Reply:
🤷 just ship it.

Good enough for a jam